src/north/CsvToHttp/CsvToHttp.class.js

```diff
@@ -174,12 +267,47 @@ class CsvToHttp extends ApiHandler {
           object.error.push(`Header "${mapping.httpField}": ${response.error}`)
         }
         object.value[field] = response.value
+      } else if (mapping.csvField.match(REGEX_CONTAIN_VARIABLE_STRING)) {
+        // split the input into a array of string
+        // "test ${value}" => [ "test ", ${value}]
+        const csvFieldSplit = mapping.csvField.split(REGEX_SPLIT_TEMPLATE_STRING).filter(Boolean)
+        const field = mapping.httpField
+        csvFieldSplit.forEach((element) => {
+          if (element.match(REGEX_MATCH_VARIABLE_STRING)) {
+            const headerToGet = element.match(REGEX_GET_VARIABLE)
+            headerToGet.forEach((header) => {
+              const response = CsvToHttp.convertToCorrectType(csvRowInJson[header], mapping.type)
+              if (response.error) {
+                object.error.push(`Header "${mapping.httpField}": ${response.error}`)
+              }
+              if (response.value) {
+                object.value[field] = CsvToHttp.insertValueInObject(object.value[field], response.value)
+              }
+            })
+          } else {
+            object.value[field] = CsvToHttp.insertValueInObject(object.value[field], element)
+          }
+        })
       }
     })
 
     return object
   }
 
+  /**
+   * @param {Mixed} object - object
+   * @param {Mixed} value - value
+   * @return {Mixed} - The converted value
+   */
+  static insertValueInObject(object, value) {
+    if (object) {
+      let response = object
+      response += value
```

Comment:
If I understand correctly, in case of aggregated field, the value is added to the previous one ? 
It may need a separator no ?
You may improve the doc of this function for better understanding